test/secretary/test/core.cljs

```diff
@@ -1,72 +1,77 @@
 (ns secretary.test.core
-  (:require [cemerick.cljs.test]
-            [secretary.core :as secretary :include-macros true :refer [defroute]])
-  (:require-macros [cemerick.cljs.test :refer [deftest is are testing]]))
+  (:require [cemerick.cljs.test :as test]
+            [secretary.core :refer
+             [encode-query-params decode-query-params route-matches
```

Comment:
I would prefer not to user `refer` here. It just makes maintenance more of a chore when things change. If it's because `secreatary` seems too long to type then I would be fine with an alias such as `s` or `sec`.